script/excludes/pl.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+return [
+    /*
+    |--------------------------------------------------------------------------
+    | Exclusion list
+    |--------------------------------------------------------------------------
+    |
+    | This is a list of exclusions for words or phrases where the original
+    | form of the word has the same spelling in a given language.
+    |
+    */
+
+    'Email',
```

Comment:
I do not agree, some languages write "E-mail", in other languages they write without a hyphen, ie "Email", and if I am not mistaken in Sweden they write "E-post" (even in google Translate it says so, but it is known that he cannot always be trusted)
Therefore, it is not the ToDo generator in a given language that gives the information that it is not translated. If the translator adds a given language to the exclusion file, it will be known that it is translated.

Reply:
I think this is a separate topic for discussion.

Reply:
Brought the question into [discussion](https://github.com/Laravel-Lang/lang/discussions/1376). Now I approve the code.